README.md

```diff
@@ -28,6 +28,15 @@ Documentation for cert-manager can be found at [cert-manager.readthedocs.io](htt
 Please make sure to select the correct version of the documentation to view on
 the bottom left of the page.
 
+For the common use-case of automatically issuing TLS certificates to
+Ingress resources, aka a [kube-lego](https://github.com/jetstack/kube-lego)
+replacement, see the [cert-manager nginx ingress quick start
+guide](docs/tutorials/quick-start/index.rst).
+
+See [Getting started](https://cert-manager.readthedocs.io/en/latest/getting-started/)
+within the [documentation](https://cert-manager.readthedocs.io/en/latest/))
```

Comment:
```suggestion
within the [documentation](https://cert-manager.readthedocs.io/en/latest/)
```

Looks like you missed the closing bracket on this change!! 😬

Reply:
Ahh!!! 🤦‍♂️  fixed!